cm-mlops/script/get-generic-python-lib/customize.py

```diff
@@ -38,8 +38,8 @@ def preprocess(i):
                 version = env.get('CM_HOST_OS_VERSION', '')
                 if version:
                     version_split = version.split(".")
-                    if (int(version_split[0]) >= 23) and ('--break-system-packages' not in extra):
-                        extra += '  --break-system-packages '
+                    if (int(pip_version[0]) >= 23) and ('--break-system-packages' not in extra):
```

Comment:
now do we need the version information of ubuntu? We can remove that part right?

Reply:
if r['return'] == 16:
            extra = env.get('CM_GENERIC_PYTHON_PIP_EXTRA','')

should we need this?

Reply:
yes, we need that

Reply:
I have not removed the code as "extra" variable was used later in the code